run

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+RUN="docker-compose run --rm"
+command=$1
+args=${@:2}
+
+case $command in
+
+sh)
+$RUN mermaid sh $args
+;;
+
+install)
+$RUN mermaid sh -c "npx pnpm install"
+;;
+
+test)
+$RUN mermaid sh -c "npx pnpm test"
+;;
+
+lint)
+$RUN mermaid sh -c "npx pnpm -w run lint:fix"
```

Comment:
Do we need `npx` after the first install?

Reply:
No, that is up to developer. As you can see, this is simply a script, which is optional to use. No other instructions on that point in CONTRIBUTING.md either